shared.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This config is shared between the default and react configs
+ */
+module.exports = {
+  plugins: [
+    "jest",
+    "jsdoc",
+    "sort-keys-fix",
+    "sort-destructure-keys",
+    "node",
+    "@typescript-eslint",
+  ],
+  rules: {
+    "brace-style": ["error", "stroustrup"],
```

Comment:
I vote we get rid of this if it's FE only

Reply:
Yup, happy with that